papers/helge_eichhorn/eichhorn.rst

```diff
@@ -0,0 +1,318 @@
+These include
+
+* the name of the body,
+* the gravitational parameter :math:`\mu`,
+* the mean radius :math:`r_m`,
+* the equatorial radius :math:`r_e`,
+* the polar radius :math:`r_p`,
+* the :math:`J_2` coefficient of the bodies gravity potential.
+
+.. * and the identification code used within the JPL ephemerides.
+
+The State Class
+~~~~~~~~~~~~~~~
+
+To define the state of a spacecraft in space-time we need the following information
+
+* the position vector (:math:`\vec{r} \in \mathbf{r^3}`),
+* the velocity vector (:math:`\vec{v} \in \mathbf{r^3}`),
+* the corresponding moment in time, the so-called epoch,
+* the reference frame in which the vectors are defined,
+* the central body which also defines the origin of the coordinate system,
+* and additional spacecraft status parameters such as mass.
+
+While the information could certainly be stored in a single Numpy-array an object-oriented programming (OOP) approach offers advantages.
+Since all necessary data can be encapsulated in the object most orbital characteristics can be calculated by calling niladic or monadic instance methods.
+Keeping the number of parameters within the application programming interface (API) very small, as recommended by Robert C. Martin [RCM08]_, improves usability, e.g. the user is not required to know the order of the function parameters.
+OOP also offers the opportunity to integrate the ``State`` class with the Python object model and the Jupyter notebook to provide rich human-friendly representations.
+
+State vectors also provide methods for backwards and forwards propagation.
+Through propagation trajectories are generated, which are instances of the ``Orbit`` class.
+
+The Orbit Class
+~~~~~~~~~~~~~~~
+
+In contrast to the ``State`` class which represents a single state in space-time the ``Orbit`` class spans a time interval and contains several spacecraft states.
+It provides all necessary tools to analyze the evolution of the trajectory over time including
+
+* quick visualizations in three-dimensional space and two-dimensional projections,
+* evolution of orbital characteristics,
+* and determination of intermediate state vectors.
+
+Exemplary Usage
+---------------
+
+In this example we use the Plyades library to conduct an analysis of the orbit of the International Space Station (ISS) [#]_.
+We obtain the inital state data on August 28, 2015, 12:00h from NASA realtime trajectory data [NAS15]_ and  use it to instantiate a Plyades ``State`` object as shown below.
+
+.. [#] A Jupyter notebook with this analysis can be obtained from `Github <https://github.com/helgee/euroscipy-2015>`_.
+
+.. code-block:: python
+
+    iss_r = np.array([
+        -2775.03475,
+        4524.24941,
+        4207.43331,
+        ]) * astropy.units.km
+    iss_v = np.array([
+        -3.641793088,
+        -5.665088604,
+        3.679500667,
+        ]) * astropy.units.km/units.s
```

Comment:
]) * astropy.units.km/astropy.units.s

Not sure though. I think the example comes from your real code, so it may be OK.